cli/schema/cypress.schema.json

```diff
@@ -253,15 +253,15 @@
           "default": false,
           "description": "Allows listening to the `before:run`, `after:run`, `before:spec`, and `after:spec` events in the plugins file during interactive mode."
         },
-        "experimentalSourceRewriting": {
+        "experimentalSessionAndOrigin": {
           "type": "boolean",
           "default": false,
-          "description": "Enables AST-based JS/HTML rewriting. This may fix issues caused by the existing regex-based JS/HTML replacement algorithm."
+          "description": "Enables cross-origin and improved session support, including the `cy.origin` and `cy.session` commands."
```

Comment:
Can we add the expected `on.cypress.io` link that will provide more information for this configuration option?

Reply:
Add it to the description? It doesn't look like we do that for any other experimental flags.

Reply:
We did in the json schema for `experimentalSessionSupport` and we do for the typescript descriptions which is why I noted it.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/b23ddf4c12f339a984f5d22a61fe27df0fb5887b